Topics/Tech_Stacks/NumPy.md

```diff
@@ -0,0 +1,199 @@
+# Learning NumPy
+
+## Table of Contents
+
+### [Introduction](#introduction-1)
+
+### [Set Up](#set-up-1)
+
+### [Array Basics](#array-basics-1)
+
+### [Matrices and Operations](#matrices-and-operations-1)
+
+### [Additional Resources](#additional-resources-1)
+
+## Introduction
+This is a short introduction to NumPy, for people looking for a basic overview of the library before doing further research into how to use it, which can be done through visiting some of the additional resources linked at the bottom of this page. Specifically, all the topics covered on this page as well as many more are discussed in both the official beginners guide and the video tutorial linked resources.
+NumPy is a Python library which provides users with new data structures which represent arrays of one or more dimensions, as well as a variety of methods for different mathematical operations which chain together effectively. It is designed for use in engineering, statistical, mathematical, as well as any other computing and/or scientific contexts where complicated operations are to be conducted on data sets of various shapes efficiently.
```

Comment:
What would be a use case in a more group project setting? How would this be applied in 301? The examples given seem to be more academic.